docs/pages/getting-started/running-docker.rst

```diff
@@ -0,0 +1,96 @@
+.. _running-docker:
+
+Getting Started on Docker
+#########################
+
+If you already use Docker, the easiest way to get your Nuts Node up and running for development or production is
+using Docker. To use the latest `master` build use `nutsfoundation/nuts-node:master` (for production environments
+it's advisable to use a specific version).
+
+First determine where you put the files required for the Nuts node, which will be mounted into the Docker container.
+
+Node TLS Certificate
+********************
+
+To connect to an existing Nuts network you need a TLS certificate which authenticate your node. For the development network
+you can use the `nuts-network-development-ca` to directly issue a certificate for your node.
+
+```
+git clone https://github.com/nuts-foundation/nuts-development-network-ca
+cd nuts-development-network-ca && ./issue-cert.sh localhost
+```
+
+The example above issues a certificate for `localhost`, replace it with the correct hostname if you want peers to be able
+to connect to your node. Rename the file to `certificate-and-key.pem`.
+
+Note that the Git repository contains the Certificate Authority certificate (`ca.pem`) which will function as truststore.
+Copy this file as `truststore.pem`.
+
+YAML Configuration File
+***********************
+
+Copy the YAML file below and save it as `nuts.yaml`:
+
+```
+datadir: /opt/nuts
+network:
+  trustStoreFile: /opt/nuts/truststore.pem
+  certFile: /opt/nuts/certificate-and-key.pem
```

Comment:
shouldn't `certificate-and-key.pem` be 2 different files?

Reply:
Not neccesarrily. My nodes run with a single, concat file (cert + pem). I updated the instructions with a `cat` command so that it properly combines the files.